src/main/drivers/vtx_common.h

```diff
@@ -28,21 +28,25 @@ typedef enum {
 
 struct vtxVTable_s;
 
+typedef struct vtxDeviceCapability_s {
+    uint8_t bandCount;
+    uint8_t channelCount;
+    uint8_t powerCount;
+} vtxDeviceCapability_t;
+
 typedef struct vtxDevice_s {
     const struct vtxVTable_s *vTable;
 
-    uint8_t numBand;
-    uint8_t numChan;
-    uint8_t numPower;
+    vtxDeviceCapability_t capability;
 
-    uint16_t *freqTable;  // Array of [numBand][numChan]
-    char **bandNames;    // char *bandNames[numBand]
```

Comment:
Not sure if character based names should make it down to the device driver level (even just as pointers to tables). Generally it would be better if they stayed in the IO layer. But probably not worth changing at this stage.